pennylane/plugins/tests/conftest.py

```diff
@@ -79,6 +79,40 @@ def _skip_if(dev, capabilities):
     return _skip_if
 
 
+@pytest.fixture(scope="session")
+def skip_if_ops():

```

Comment:
Minor point: could we pass the `device` fixture here?

Reply:
I am not sure if this will work, or if the same problem occurs: the device is only created inside a test, because it may be created with different wire classes that are specified by the fixture.